tests/test_ru_example.py

```diff
@@ -0,0 +1,69 @@
+import unittest
+
+from wikitextprocessor import Wtp
+
+from wiktextract.config import WiktionaryConfig
+from wiktextract.extractor.ru.example import process_example_template
+from wiktextract.extractor.ru.models import Sense
+from wiktextract.wxr_context import WiktextractContext
+
+
+class TestRUExample(unittest.TestCase):
+    def setUp(self) -> None:
+        self.wxr = WiktextractContext(
+            Wtp(lang_code="ru"),
+            WiktionaryConfig(dump_file_lang_code="ru"),
+        )
+
+    def tearDown(self) -> None:
+        self.wxr.wtp.close_db_conn()
+
+    def get_default_sense_data(self) -> Sense:
+        return Sense()
+
+    def test_ru_extract_example(self):
+        test_cases = [
+            # Ignores empty template
+            {"input": "{{пример|}}", "expected": []},
+            # https://ru.wiktionary.org/wiki/Красная_Шапочка
+            {
+                "input": "{{пример|Недолго думая, отправляю овощ в рот.|М. И. Саитов|Островки||Бельские Просторы|2010|источник=НКРЯ}}",
+                "expected": [
+                    {
+                        "ref": {
+                            "author": "М. И. Саитов",
+                            "collection": "Бельские Просторы",
+                            "date_published": "2010",
+                            "source": "НКРЯ",
+                            "title": "Островки",
+                        },
+                        "text": "Недолго думая, отправляю овощ в рот.",
+                    }
+                ],
+            },
+            # https://ru.wiktionary.org/wiki/house
+            {
+                "input": "{{пример|This is my {{выдел|house}} and my family’s ancestral home.||перевод=Это мой {{выдел|дом}} и поселение моих семейных предков.}}",
```

Comment:
Don't use template:
```suggestion
                "input": "{{пример|This is my house and my family’s ancestral home.||перевод=Это мой дом и поселение моих семейных предков.}}",
```

or add a simple "выдел" template with body `{{{1|}}}`.